airflow/providers/airbyte/sensors/airbyte.py

```diff
@@ -80,35 +81,24 @@ def __init__(
         self.airbyte_conn_id = airbyte_conn_id
         self.airbyte_job_id = airbyte_job_id
         self.api_version = api_version
-        self.api_type = api_type
+        self.skip_policy = SkipPolicy.SKIP_ON_SOFT_ERROR
```

Comment:
I encountered a few errors while updating the tests for sensors requesting to add the `skip_policy`. Since I don't use Airflow frequently and don't contribute much, I need help to ensure if this is.